Release failed

With logs:

```
Run rubygems/release-gem@v1
Run git config --global user.email "$(git log -1 --pretty=format:'%ae')"
Run git credential-cache --timeout=300 store <<EOF
Run rubygems/configure-rubygems-credentials@v1.0.0
  
Run bundle exec rake release
  
rake aborted!
Don't know how to build task 'release' (See the list of available tasks with `rake --tasks`)
/home/runner/work/altertable-ruby/altertable-ruby/vendor/bundle/ruby/3.4.0/gems/rake-13.3.1/exe/rake:27:in '<top (required)>'
/opt/hostedtoolcache/Ruby/3.4.8/x64/bin/bundle:25:in 'Kernel#load'
/opt/hostedtoolcache/Ruby/3.4.8/x64/bin/bundle:25:in '<main>'
(See full trace by running task with --trace)
Error: Process completed with exit code 1.
Run git credential-cache exit || true
```

--- a/Rakefile
+++ b/Rakefile
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "rspec/core/rake_task"
+require "bundler/gem_tasks"
 
 RSpec::Core::RakeTask.new(:spec)
 

